packages/core/src/rpc/sync.ts

```diff
@@ -1,19 +1,20 @@
 import { createHash } from "node:crypto"
+import assert from "node:assert"
 
 import chalk from "chalk"
 import type { Duplex } from "it-stream-types"
 import type { Uint8ArrayList } from "uint8arraylist"
 
 import * as okra from "node-okra"
 
-import { toHex } from "../utils.js"
 import { BinaryMessage, decodeBinaryMessage } from "../encoding.js"
 import { Client } from "./client.js"
 
 export async function sync(
 	mst: okra.Tree,
 	stream: Duplex<Uint8ArrayList, Uint8ArrayList | Uint8Array>,
-	applyBatch: (messages: [Buffer, BinaryMessage][]) => Promise<void>
+	handleMessage: (hash: Buffer, data: Uint8Array, message: BinaryMessage) => Promise<void>
+	// applyBatch: (messages: [hash: Buffer, data: Uint8Array, message: BinaryMessage][]) => Promise<void>
```

Comment:
Do we want to eventually delete this line or just comment it out?

Reply:
ah good catch yeah I left a few commented lines in there I meant to delete